index.js

```diff
@@ -1,7 +1,15 @@
 'use strict';
 const Build = require('./lib/build');
+const factory = require('./lib/factory');
 const Job = require('./lib/job');
 const Pipeline = require('./lib/pipeline');
 const User = require('./lib/user');
 
-module.exports = { Build, Job, Pipeline, User };
+module.exports = {
+    Build,
+    configureBuildModel: factory.configureBuildModel,
```

Comment:
If every model had a factoryMethod, what would the difference be between how we're doing it now?

Adding it to the prototype would be us using another method instead of the constructor. You're still exposing the user to invoke `new`, which doesn't yield us the benefits of centralizing the configuration in one call.